python/Ganga/Runtime/bootstrap.py

```diff
@@ -1069,12 +1070,12 @@ def convert_merger_to_postprocessor(j):
 
         # JobTree
         from Ganga.Core.GangaRepository import getRegistry
-        jobtree = GPIProxyObjectFactory(getRegistry("jobs").getJobTree())
+        jobtree = getRegistry("jobs").getJobTree()
```

Comment:
I'm wanting to move away from ```GPIProxyObjectFactory``` to just use ```addProxy``` since we're now allowing for users to add a proxy to a class. Hence the changes which started this.
I think the ```addProxy``` should eventually cope with adding proxies to objects and classes atm but it should make dealing with adding/removing proxies a lot more transparent (I hope)